Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -278,10 +278,10 @@ enum GSError {
         let loggable = false
     }
 
-    struct ENSAddressNotFound: DetailedLocalizedError {
+    struct BlockhainAddressNotFound: DetailedLocalizedError {
```

Comment:
sorry, one more thing that I overlooked - let's revert this struct (and `howToFix` below, too)

Reply:
What a great catch! I completely forgot to rename it back. thank you!